src/contentmanagerside.cpp

```diff
@@ -92,7 +93,14 @@ void ContentManagerSide::setContentManager(ContentManager *contentManager)
 
 void ContentManagerSide::setCategories(QStringList categories)
 {
-    mp_categories->setSelections(categories, KiwixApp::instance()->getSettingsManager()->getCategoryList());
+    const auto lang = QLocale().name().split('_')[0].toStdString();
+    KiwixChoiceBox::SelectionList sList;
+    for (const auto& category : categories) {
+        const auto tranlation = QString::fromStdString(kiwix::translateBookCategory(lang, category.toStdString()));
+        sList.append({category, tranlation});
+    }
+    
+    mp_categories->setSelections(sList, KiwixApp::instance()->getSettingsManager()->getCategoryList());
```

Comment:
This is indeed a problem. Since the getCategoryList is also called elsewhere, I believe the solution will be to re-translate in settingsmanager when we first load the filter from the settings.

Reply:
Done